x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/application/rules/QueryRuleTests.java

```diff
@@ -170,7 +157,8 @@ public void testApplyRuleWithOneCriteria() {
             randomAlphaOfLength(10),
             QueryRule.QueryRuleType.PINNED,
             List.of(new QueryRuleCriteria(EXACT, "query", List.of("elastic"))),
-            Map.of("ids", List.of("id1", "id2"))
+            Map.of("ids", List.of("id1", "id2")),
+            randomBoolean() ? randomIntBetween(0, 100) : null
```

Comment:
can we use `MAX_PRIORITY`? if not the constant itself, then the value of `1000000`?

Reply:
Sure, I can update